.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Maybe we can add a `setup-node` below without specifying node version, so that it can still install problem matcher. In this way, it can annotate eslint warnings and ts errors on the pull request itself, which can let reviewer spot newly added eslint warnings more easily.

See https://github.com/actions/setup-node#v2

Reply:
i see thanks, didn't know about problem matchers 😅 

https://github.com/actions/toolkit/blob/master/docs/problem-matchers.md#problem-matchers

Reply:
will add this everywhere 🤪